src/VisualStudio/Core/Test.Next/Remote/JsonConverterTests.cs

```diff
@@ -139,6 +139,15 @@ public void TestPinnedSolutionInfo()
             });
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.RemoteHost)]
+        public void TestAnalyzerPerformanceInfo()
+        {
+            VerifyJsonSerialization(
+                new AnalyzerPerformanceInfo("testAnalyzer", builtIn: false, timeSpan: TimeSpan.FromMilliseconds(12345)),
```

Comment:
>12345 [](start = 112, length = 5)

is this random?

Reply:
yes. it just make sure value persevered across json conversions.